litex/soc/cores/spi_xip/common.py

```diff
@@ -0,0 +1,8 @@
+spi_phy_ctl_layout = [
```

Comment:
Please add comments here.

Reply:
I've added some comments to the version in litex-hub/litespi#2